Xamarin.Forms.Sandbox.iOS/SkiaSharpButtonRenderer.Shared.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System.ComponentModel;
+using SkiaSharp;
+using Xamarin.Forms;
+using System.Collections.Generic;
+
+#if __ANDROID__
+using SkiaSharp.Views.Android;
+using SKPaintSurfaceEventArgs = SkiaSharp.Views.Android.SKPaintSurfaceEventArgs;
+#elif __IOS__
+using SkiaSharp.Views.iOS;
+using SKPaintSurfaceEventArgs = SkiaSharp.Views.iOS.SKPaintSurfaceEventArgs;
+#endif
+
+[assembly: ExportRenderer(typeof(Button), typeof(SkiaSharpVisual.SkiaSharpButtonRenderer), new[] { typeof(SkiaSharpVisual.SkiaSharp) })]
```

Comment:
It appears that the only extra thing to use visual is the last parameter:
```csharp
new[] { typeof(SkiaSharpVisual.SkiaSharp) }
```

Which is how this was all designed. So, everything is working as intended.